DeepSpeech.ipynb

```diff
@@ -98,6 +98,10 @@
     "ds_importer = os.environ.get('ds_importer', 'ted')\n",
     "ds_dataset_path = os.environ.get('ds_dataset_path', os.path.join('./data', ds_importer))\n",
     "\n",
+    "ds_limit_train = int(os.environ.get('ds_limit_train', 0))\n",
+    "ds_limit_dev   = int(os.environ.get('ds_limit_dev',   0))\n",
+    "ds_limit_test  = int(os.environ.get('ds_limit_test',  0))\n",
```

Comment:
Could you move these into the **Global Constants** section? Just because most tunable parameters are there.